testsuite/python/constant_pH.py

```diff
@@ -55,6 +55,9 @@ def test_ideal_alpha(self):
             default_charges={type_HA: 0, type_A: -1, type_H: +1})
         RE.constant_pH = pH
 
+        # Set the hidden particle type to the lowest possible number to speed up the simulation
+        RE.set_non_interacting_type(3)
```

Comment:
Consider making use of a types list or len(default_charges.keys())

Reply:
or `max(default_charges)` which returns the largest dict key